site/_docs/datafiles.md

```diff
@@ -114,3 +114,33 @@ The organizations can then be accessed via `site.data.orgs`, followed by the fil
 </ul>
 {% endraw %}
 {% endhighlight %}
+
+## Example: Accessing a specific author
+
+Pages and posts can also access a specific data item. The example below shows how to access a specific item:
+
+`_data/people.yml`:
+{% highlight yaml %}
+dave:
+    name: David Smith
+    twitter: DavidSilvaSmith
+{% endhighlight %}
+
+The author can then be specified as a page variable in a post's frontmatter:
+
+{% highlight html %}
+{% raw %}
+---
+title: sample post
+author: dave
+---
+
+{% assign author = site.data.people.[page.author] %}
```

Comment:
It's a weird syntax (super useful though, took me some googling to get it). I confirmed it's working correctly on my site: https://github.com/davidsilvasmith/bitcoinbulls.net/blob/gh-pages/_layouts/post.html

Sorry about including the LSI docs. I'm new to pull requests. I'll be better with future pulls :)